src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Microsoft.AspNetCore.Razor.Test.Common.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
```

Comment:
There isn't a `Microsoft.CodeAnalysis.Testing.Verifiers.XUnit` that brings in a new enough version of NuGet client dlls to not get conflicts with older Newtonsoft versions. Therefore, pinning to latest.